docs/docs/api/endpoints/components.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Components
+sidebar_label: Components
+---
+
+The `components.*` API is about controlling the individual components which are running inside `Airy Core`.
+
+## Get
+
+Retrieve the configuration of the components currently started in `Airy Core`.
+
+`POST /components.get`
+
+The list of the currently configured components is returned.
+
+**Sample response**
+
+```json5
+{
+  "security": {
+    "systemToken": "token",
+    "allowedOrigins": "*",
+    "jwtSecret": "secret"
+  },
+  "components": {
+    "sources": {
+      "facebook": {
+        "appId": "id",
+        "appSecret": "secret",
+        "webhookSecret": "secret"
+      },
+      "google": {
+        "partnerKey": "key",
+        "saFile": "file"
+      }
+    }
+  }
+}
+```
+
+## Update
+
+Update the configuration of a list of components.
+
+`POST /components.update`
+
+**Sample request**
+
+```json
+{
+  "Security": {
```

Comment:
Payload needs to follow our `snake_case` convention

Reply:
> Payload needs to follow our `snake_case` convention

Thanks :+1:. I changed it for `security` and `data`. For the values which were in the `airy.yaml` (`allowedOrigins`, `systemToken`, `jwtSecret`, ...) I suggest we do it in a subsequent PR, as it has lots of changes across almost all components.